hapi/src/api/account.api.js

```diff
@@ -569,17 +569,26 @@ const transfer = async (from, details) => {
 
   const newBalance = await lifebankcoinUtils.getbalance(details.to)
   await historyApi.insert(transaction)
-  await notificationApi.insert({
-    account: details.to,
-    title: 'New tokens',
-    description: `From ${from} ${details.memo}`,
-    type: 'new_tokens',
-    payload: {
-      currentBalance,
-      newBalance,
-      transaction: transaction.transaction_id
-    }
-  })
+
+  if (notification) {
+    notification.payload.currentBalance = currentBalance
+    notification.payload.newBalance = newBalance
+    notification.payload.transaction = transaction.transaction_id
+    await notificationApi.insert(notification)
+  } else {
+    await notificationApi.insert({
+      account_from: from,
+      account_to: details.to,
+      title: 'New tokens',
+      description: `From ${from} ${details.memo}`,
```

Comment:
Shouldn't this be in both languages?

Reply:
I don't think so, it is for backend logic purpose but I'm not sure at all